music21/metadata/__init__.py

```diff
@@ -582,95 +1255,6 @@ def search(self, query=None, field=None, **kwargs):
                     return True, innerField
         return False, None
 
-    def setWorkId(self, idStr, value):
```

Comment:
can we mark this as `@deprecated`, remove docs, and just call `__setattr__` instead?  It seems rather easy to give this a one-year deprecation cycle.  I know I've used it elsewhere.

Reply:
Yes.  I was wondering about that.

Reply:
Done.